src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,60 @@
+import { useSearchParams } from 'react-router-dom';
+import { useCallback } from 'react';
+import classNames from 'classnames';
+import { SearchLink } from './SearchLink';
+
+const CENTURIES = [16, 17, 18, 19, 20];
+const SEX = {
+  All: null,
+  Male: 'm',
+  Female: 'f',
+};
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const getCurrentCenturies = useCallback((century: string) => {
+    const centuriesFromSearch = searchParams.getAll('centuries');
+
+    if (centuriesFromSearch === null) {
+      return [century];
+    }
```

Comment:
```suggestion
    if (!centuriesFromSearch) {
      return [century];
    }
```

should work same but simplify